test/runtests.jl

```diff
@@ -1,6 +1,8 @@
 using Measurements, SpecialFunctions, QuadGK, Calculus
 using Test, LinearAlgebra, Statistics, Unitful, Printf, Aqua
-Aqua.test_all(Measurements)
+
+
+#Aqua.test_all(Measurements)
```

Comment:
Why?

Reply:
ohh, sorry, that wasn't meant to be in the PR, but Aqua has problems in Formatting extensions ([#105](https://github.com/JuliaTesting/Aqua.jl/issues/105), ([#107](https://github.com/JuliaTesting/Aqua.jl/issues/107)

Reply:
Doesn't look like https://github.com/PainterQubits/Unitful.jl/pull/622 had to remove this.  Can it be brought back?  It's pointless to have Aqua has a test dependency and don't use it (and I hate commented out code: code is either used or it doesn't exist)